packages/plugin-ext-vscode/src/node/plugin-vscode-extension.ts

```diff
@@ -0,0 +1,42 @@
+/********************************************************************************
+ * Copyright (C) 2022 Red Hat, Inc.
```

Comment:
I am not but since I only moved classes to a new file I thought I should keep the original contributor company. However, with the latest changes this no longer applies since I moved the vscode extension code to the plugin anyway. If that should change, I will keep that in mind next time, sorry for that!